src/mono/wasm/debugger/debugger.md

```diff
@@ -0,0 +1,23 @@
+# Mono Wasm Debugger
+
+## Native debugging
+
+It's possible to debug native code and managed code using BrowserDebugProxy.
+
+Steps:
+- Install C/C++ DevTools Support (DWARF) extension on chrome.
```

Comment:
Is there a link that can be provided for this? How does one find this tool.

Reply:
Do you mean like this?
https://chrome.google.com/webstore/detail/cc%20%20-devtools-support-dwa/pdcpmagijalfljmkmjngeonclgbbannb